app/assets/javascripts/search.js

```diff
@@ -31,7 +31,7 @@
       return searchResultData;
     },
     enableLiveSearchCheckbox: function ($searchResults) {
-      if ($searchResults.length > 0) {
+      if ($searchResults.length >= 0) {
```

Comment:
Thank you Tommy for the feedback and thank you Kush for closing the PR. I've updated the commit message, please let me know if it is still not clear.